fem/config/loki-config.yml

```diff
@@ -3,32 +3,25 @@ auth_enabled: false
 server:
   http_listen_port: 3100
 
-ingester:
-  wal:
-    dir: /tmp/wal
-  lifecycler:
-    address: 0.0.0.0
-    ring:
-      kvstore:
-        store: inmemory
-      replication_factor: 1
-    final_sleep: 0s
-  chunk_idle_period: 5m
-  chunk_retain_period: 30s
+common:
+  ring:
+    instance_addr: 127.0.0.1
+    kvstore:
+      store: inmemory
+  replication_factor: 1
+  path_prefix: /tmp/loki
 
 schema_config:
   configs:
     - from: 2020-05-15
-      store: boltdb
+      store: tsdb
       object_store: filesystem
-      schema: v11
+      schema: v13
       index:
         prefix: index_
-        period: 168h
+        period: 24h
```

Comment:
After taking a closer look at the documentation, I figured out that this configures the (daily) rotation of the index file, but not the retention time nor an automatically data maintenance (of the index).
To avoid the 'inmemory database' running out of space, we should configure a data retention.
That's done by using 'compactor': https://grafana.com/docs/loki/latest/operations/storage/retention/#retention-configuration

Reply:
We are using no inmemory storage - anyway, deleting unused data is always a good idea.